Yank/analyze.py

```diff
@@ -199,6 +199,9 @@ def __init__(self, reporter, name=None, reference_states=(0, -1)):
                 O[j] is not used
             For O where differences between states are required (e.g. Free Energy):
                 O[i,j] = O[j] - O[i]
+        analysis_kwargs : None or dict, optional
+            Dictionary of extra keyword arguments to pass into the analysis tool, typically MBAR.
+            For instance, the initial guess of weights to give to MBAR would be something like {'f_k':[0,1,2,3]}
```

Comment:
You're right, "weights" is imprecise and confusing since the weights are per-sample. I'll update this everywhere

Most the variable names and defaults in general I chose for this could use some feedback, so I'm happy to fix any that dont sound right